examples/web-chat/src/config.ts

```diff
@@ -0,0 +1,13 @@
+import { Protocols } from "@waku/interfaces";
+
+export const CONTENT_TOPIC = "/toy-chat/2/huilong/proto";
+
+const PUBLIC_KEY = "AOGECG2SPND25EEFMAJ5WF3KSGJNSGV356DSTL2YVLLZWIV6SAYBM";
+const FQDN = "test.waku.nodes.status.im";
+export const ENR_TREE = `enrtree://${PUBLIC_KEY}@${FQDN}`;
```

Comment:
This does not need to be 3 variables

Reply:
indeed, but keeping it like this makes it clear what is what for the example purpose